ios/PolyPodApp/PodApi/PodApiError.swift

```diff
@@ -21,6 +25,14 @@ extension PodApiError: LocalizedError {
             return "Parameter missing"
         case .noSuchFile(let path):
             return "No such file: \(path)"
+        case .noAppDelegate:
+            return "AppDelegate not available"
+        case .badArgumentData(let arg):
+            return "Bad argument data: \(arg)"
+        case .failedToReadGraph:
+            return "Failed to read graph"
+        case .badData:
+            return "Bad data"
```

Comment:
Don't you have more information here? I gather this is going to bubble up to the user, could there be a friendlier way to let them know this? (Also: more informative)

Reply:
Maybe add an argument as above?

Reply:
Hm, I suppose I could, good point! Not sure when we're gonna touch this again, so better add more details now.